nss-day-cohort-41/tabloidcli-kermit-krew
# Input verification and cleanup

# Description
Implemented correct input verification for new Authors, Journal entries, and Blogs. This will prevent crashes upon improper user input. Added additional Console.Clear() statements throughout the app to help keep the text within the console window, which will help maintain the background color.
Fixes - No tickets
## Type of change
Please delete options that are not relevant.
# Testing Instructions
From TabloidCLI, run the program. Add new items to Authors, Journal, and Blogs. A blank input in any of the above will trigger re-issuing the prompt. Change the background color and make sure it is maintained across all pages, especially after erroneous input in the above.
# Checklist:
